ingest/src/main/java/com/microsoft/azure/kusto/ingest/exceptions/IngestionClientException.java

```diff
@@ -18,6 +18,10 @@ public IngestionClientException(String message, Exception exception) {
         super(message, exception);
     }
 
+    public IngestionClientException(String message, Throwable throwable) {
```

Comment:
This is just a more general version of the prev ctor, so you can remove that one